cohorts/load.py

```diff
@@ -80,6 +86,18 @@ def __init__(self,
         self.nonsynonymous_effect_cache_name = "cached-nonsynonymous-effects"
         self.neoantigen_cache_name = "cached-neoantigens"
 
+    def verify_survival(self):
```

Comment:
This doesn't allow OS == PFS and it often will be when the values have been censored.